src/site/components/error-message.md-njk

```diff
@@ -0,0 +1,39 @@
+---
+layout: layouts/markdown-content-page.njk
+title: Error message
+description: When there is a validation error, use an error message to explain what went wrong and how to fix it.
+tags: components
+visual_regression:
+  selector: main
+---
+
+{%- from "components/component-preview/macro.njk" import guidanceComponentPreview -%}
+
+Show an error message next to the field and in the error summary when there is a validation error.
+
+{{ guidanceComponentPreview({
+  componentName: "details",
```

Comment:
Looks like this is the details component :smile: 

There is the actual error message component but probably it should be the date input error example like https://design-system.service.gov.uk/components/error-message/?

Reply:
Ahh copy past error. Will pull in the date input component with error